flow/chain.go

```diff
@@ -41,14 +41,17 @@ func (c *Chain) Run(ctx context.Context, prompt *blades.Prompt, opts ...blades.M
 // RunStream executes the chain of runners sequentially, streaming the output of the last runner.
 func (c *Chain) RunStream(ctx context.Context, prompt *blades.Prompt, opts ...blades.ModelOption) (blades.Streamer[*blades.Generation], error) {
```

Comment:
For reference only, I feel it would be better to let the user handle the `Close` action themselves.